components/CollectivePicker.js

```diff
@@ -302,6 +302,7 @@ class CollectivePicker extends React.PureComponent {
               ref={mergeRefs([this.containerRef, ref])}
             >
               <StyledSelect
+                inputId="collective-picker"
```

Comment:
We should avoid doing that since there can be multiple collective pickers on the page. It's up to the places that use `CollectivePicker` to define an `inputId`.